style.css

```diff
@@ -0,0 +1,8 @@
+body {
+  background-color: #8a2be2;
+}
+
+h1 {
```

Comment:
You can use class attributes here too.

When you do it, you will be coding like a pro! 👨‍💼

Reply:
Good!